k8s/secret-challenge-vault-deployment.yml

```diff
@@ -19,6 +19,20 @@ spec:
     type: RollingUpdate
   template:
     metadata:
+      annotations:
+        vault.hashicorp.com/agent-inject: "true"
+        vault.hashicorp.com/tls-skip-verify: 'true'
+        vault.hashicorp.com/agent-inject-status: "update"
+        vault.hashicorp.com/namespace: "default"
+        vault.hashicorp.com/log-level: debug
+        vault.hashicorp.com/agent-inject-secret-challenge46: "secret/data/secret-challenge"
```

Comment:
Can we change this to a new entry with a new value please? And can you update https://github.com/OWASP/wrongsecrets/blob/master/scripts/install-vault.sh and https://github.com/OWASP/wrongsecrets/blob/master/k8s-vault-minkube-start.sh to contain that new entry, including the policy in both so that the app is allowed to use it?

Reply:
done